src/coreclr/src/utilcode/clrconfig.cpp

```diff
@@ -418,11 +381,5 @@ REGUTIL::CORConfigLevel CLRConfig::GetConfigLevel(LookupOptions options)
     if(CheckLookupOption(options, IgnoreEnv) == FALSE)
         level = static_cast<REGUTIL::CORConfigLevel>(level | REGUTIL::COR_CONFIG_ENV);
 
-    if(CheckLookupOption(options, IgnoreHKCU) == FALSE)
```

Comment:
Is this going to change behavior? It looks like we have been always reading the config from registry before this change, but it will no longer happen after this change.

It may be a good idea to remove reading of the config from registry for CoreCLR (it is reading it from the same place as .NET Framework that is a problem on its own), but it should be done as intentional change.

Reply:
Do you mean to not remove this clause in this PR? But then wouldn't it fail when building due to `IgnoreHKCU` not being defined anymore?

Reply:
You would need to remote the `if` condition, but keep the code that executed when the condition was true.

If you would like to delete the config reading from the registry, it would be fine with me too. It would need to be marked as breaking change and the PR should delete all code related to it.

Reply:
By all related code you mean find all calls to `CLRConfig::GetConfigLevel` where `options` is equal to any of the removed flags? Another question I have is what does remoting the `if` condition mean in this context?

Reply:
> what does remoting the if condition mean in this context?

Sorry, this typo - fixed above. I meant to say "You would need remove ...".

> all related code

I meant delete `REGUTIL::COR_CONFIG_USER`, `REGUTIL::COR_CONFIG_MACHINE` and all code that becomes unreachable.